base_layer/wallet/src/output_manager_service/service.rs

```diff
@@ -1159,6 +1180,119 @@ where
         Ok((tx_id, tx, fee, utxos_total_value))
     }
 
+    async fn fetch_outputs_from_node(
```

Comment:
I'm sure this function already exists?

Reply:
I think it was removed when refactored the wallet to use streaming and the new validation 
This is currently the only function that calls the base_node function: `fn fetch_matching_utxos` outside of the base_node. There are other functions that get utxo's, but none that only requests them on the hash,